test/middleware/PostQuery.test.ts

```diff
@@ -28,25 +30,24 @@ describe('Test API key header verification', () => {
             complexity: 3,
             timestamp: 100,
             tokens: -1,
+            success: true,
+            loggedOn: 0,
         };
-        postQuery = new PostQuery(mockURI, mockProjectID, mockQueryData);
-        const newPost = () => postQuery.post();
+        const newPost = () => postQuery(mockURI, mockProjectID, mockQueryData);
 
         await expect(newPost).rejects.toThrow(`[gatelog] Query data cannot be negative.`);
     });
 
     test('Error thrown when resulting projectID of query does not match the one entered', async () => {
-        postQuery = new PostQuery(mockURI, mockProjectID, mockQueryData);
-        const newPost = () => postQuery.post();
+        const newPost = () => postQuery(mockURI, mockProjectID, mockQueryData);
 
         await expect(newPost).not.toThrow(
             `[gatelog] GraphQL error, resulting query's projectID does not match the one entered`
         );
     });
 
     test('No error thrown when query data is correct syntax', async () => {
```

Comment:
This is the same test as above but your but your not expecting any errors rather than a specific one. This test covers the above one, you don't need both.

Reply:
fixed